### ISSUE TITLE
Access Gradient responsible for current update inside Layer

### ISSUE THREAD

**Issue by felipefariax** [fchollet/keras#1075]
Hello!

I would like to know if there is someway to get the grad inside some layer method during backprop, for example:

If I have a Dense Layer, do I have a method to look the grads that were responsible for the changes on the Weights of Dense Layer during an update?

Thank you@

**Comment by EderSantana** [comment]
I don't think Keras has a ready made solution for that. You would have to define your own Theano function and calculate the values inside a callback.

**Comment by felipefariax** [comment]
Thanks @EderSantana!
